docs/source/task_summary.rst

```diff
@@ -709,13 +725,12 @@ following array should be the output:
 Summarization
 ----------------------------------------------------
 
-Summarization is the task of summarizing a text / an article into a shorter text.
+Summarization is the task of summarizing a document or an article into a shorter text.
 
 An example of a summarization dataset is the CNN / Daily Mail dataset, which consists of long news articles and was created for the task of summarization.
-If you would like to fine-tune a model on a summarization task, you may leverage the ``examples/summarization/bart/run_train.sh`` (leveraging pytorch-lightning) script.
+If you would like to fine-tune a model on a summarization task, you may leverage the ``examples/summarization/bart/run_train.sh`` (XXX: wrong script) (pytorch-lightning) script.
```

Comment:
@sshleifer There is no script there indeed. Do you know where the summarization examples are? Didn't find any named like this in the seq2seq folder.

Reply:
https://github.com/huggingface/transformers/blob/72add6c98f2c0607f088fa0c78d40f11e2efa4c4/examples/seq2seq/README.md#L99

MT+Summary are both in `examples/seq2seq/finetune.py`, would be good to link to that README which has info on how to use your own data/how to distill etc.